acceptance-tests/src/test/resources/features/UKDrivingLicence/DLProveYourIdentityFullJourneyRoute.feature

```diff
@@ -120,6 +122,7 @@ Feature: Prove Your Identity Full Journey
     Given I select the radio option UK driving licence and click on Continue
     And I should be on `Who was your UK driving licence issued by` page
     And I click on DVLA radio button and Continue
+    And I should be on `Enter your details exactly as they appear on your UK driving licence – Prove your identity – GOV.UK` page
```

Comment:
Should we have this page check in twice. If the new one is right should the old one be removed?